app/konnect-audit-logs.md

```diff
@@ -0,0 +1,332 @@
+---
+title: "{{site.konnect_short_name}} audit logs"
+content_type: reference
+layout: reference
+
+products:
+    - gateway
+
+description: Use {{site.konnect_short_name}} audit logs to help you detect and respond to potential security incidents when they occur.
+
+related_resources:
+  - text: "Set up an audit log webhook for a {{site.konnect_short_name}} org"
+    url: /how-to/set-up-konnect-org-audit-log-webhook/
+  - text: "Set up an audit log webhook for Dev Portal"
+    url: /how-to/set-up-dev-portal-audit-log-webhook/
+  - text: "{{site.konnect_short_name}} Org audit log API"
+    url: /api/konnect/audit-logs/v2/
+  - text: "Dev Portal audit log API"
+    url: /api/konnect/portal-management/v2/#/operations/update-portal-audit-log-replay-job
+  - text: "{{site.base_gateway}} audit logs"
+    url: /gateway/audit-logs/
+  - text: "{{site.base_gateway}} logs"
+    url: /gateway/logs/
+---
+
+Audit logs can help you detect and respond to potential security incidents when they occur. Monitoring audit logs proactively can reduce the risk of outages and ensure continuous service for your users. No system can ever be completely secure, but audit logs can be a key part of your incident prevention infrastructure.
+
+Audit logging provides the following benefits:
+* **Security**: System events can be used to show abnormalities to be investigated, forensic information related to breaches, or provide evidence for compliance and regulatory purposes.
+* **Compliance**: Regulators and auditors may require audit logs to confirm whether certain certification standards are met.
+* **Debugging**: Audit logs can help determine the root causes of efficiency or performance issues.
+* **Risk management**: Prevent issues or catch them early.
+
+{:.warning}
+> Org-wide {{site.konnect_short_name}} audit logs are set up and managed separately from Dev Portal audit logs.
+
+## Configure audit logging
+
+Audit logging is configured via a webhook with a SIEM provider of your choice. For more information, see [Set up an audit log webhook for a {{site.konnect_short_name}} org](/how-to/set-up-konnect-org-audit-log-webhook/) and [Set up an audit log webhook for Dev Portal](/how-to/set-up-dev-portal-audit-log-webhook/).
+
+{{site.konnect_short_name}} captures three types of events:
+
+| Event type | Org audit logs | Dev Portal audit logs |
+| ---------- | ---------- | ---------- |
+| Authentication | This is triggered when a user attempts to log into the {{site.konnect_short_name}} web application or use the {{site.konnect_short_name}} API via a personal access token. Also triggered when a system account access token is used. | Triggered when a user logs in to the Dev Portal. |
+| Authorization | Triggered when a permission check is made for a user or system account against a resource. | Not currently supported |
+| Access logs | Triggered when a request is made to the {{site.konnect_short_name}} API. | Not currently supported |
+
```

Comment:
```suggestion
| Event type | Org audit logs |
| ---------- | ---------- | 
| Authentication | This is triggered when a user attempts to log into the {{site.konnect_short_name}} web application or use the {{site.konnect_short_name}} API via a personal access token. Also triggered when a system account access token is used. |
| Authorization | Triggered when a permission check is made for a user or system account against a resource. | 
| Access logs | Triggered when a request is made to the {{site.konnect_short_name}} API. | 

```

Reply:
I think I did this right. I removed the dev portal stuff. Outside of the dev portal landing page the dev portal stuff is on the michael migrate list